composer.json

```diff
@@ -6,21 +6,20 @@
 	"minimum-stability": "dev",
 	"require": {
 		"php": ">=7.4",
+		"ext-json": "*",
 		"automattic/jetpack-autoloader": "^2.4",
 		"automattic/jetpack-config": "^1.4",
 		"automattic/jetpack-connection": "^1.40",
-		"google/apiclient": "^2.12",
-		"google/apiclient-services": "~0.258",
-		"googleads/google-ads-php": "^16.0",
+		"google/apiclient": "^2.13",
+		"google/apiclient-services": "~0.286",
+		"googleads/google-ads-php": "^17.0",
```

Comment:
This looks weird, because version 17.0.0 only added support for the V11.1 API. We explicitly need 17.1.0 to support V12.

Not a problem since it did pick the latest when you upgraded:
```
- Upgrading googleads/google-ads-php (v16.0.0 => v17.1.0)
```

Reply:
Fixed in https://github.com/woocommerce/google-listings-and-ads/pull/1875/commits/b7e54aa2b468eb5d2fc51702d12e1e77bfd8340a